dbusmock/templates/systemd-base.py

```diff
@@ -28,32 +29,13 @@ def load_base(mock, _parameters):
     mock.next_job_id = 1
     mock.units = {}
 
-    mock.AddMethod(MAIN_IFACE,
-                   'StartUnit',
-                   'ss', 'o',
-                   'ret = self.StartUnit(self, *args)')
-    mock.AddMethod(MAIN_IFACE,
-                   'StartTransientUnit',
-                   'ssa(sv)a(sa(sv))', 'o',
-                   'ret = self.StartTransientUnit(self, *args)')
-    mock.AddMethod(MAIN_IFACE,
-                   'StopUnit',
-                   'ss', 'o',
-                   'ret = self.StopUnit(self, *args)')
-    mock.AddMethod(MAIN_IFACE,
-                   'GetUnit',
-                   's', 'o',
-                   'ret = self.GetUnit(self, *args)')
+    mock.AddServiceMethod(MAIN_IFACE, 'StartUnit', StartUnit)
```

Comment:
As this commit now actually adds the decorations, I wonder why you need these calls at all now? The decorated methods are supposed to "just work".

Reply:
As mentioned, when the template is `systemd-user.py` but all the implementation is in `systemd-user.py`, I didn't get any D-Bus methods to work using decorations.